.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "cSpell.words": [
+    "Dotenv",
+    "initializers",
+    "ohdear",
+    "Ohdear",
+    "sidekiq",
+    "Sidekiq"
+  ]
+}
```

Comment:
These are local config files, they should be gitignored

Reply:
no, it's global config for everyone, so that you don't have warnings on spellings if you get the project for the first time and use the spell check plugin ;)

Reply:
if you want to keep those config local you save it to user settings and not workspace config(this file)